core/src/main/java/com/sequenceiq/cloudbreak/converter/UserProfileToJsonConverter.java

```diff
@@ -26,6 +41,18 @@ public UserProfileResponse convert(UserProfile entity) {
         }
         Map<String, Object> map = entity.getUiProperties().getMap();
         userProfileResponse.setUiProperties(map == null ? new HashMap<>() : map);
+        userProfileResponse.setDefaultImageCatalog(getImageCatalog(entity));
         return userProfileResponse;
     }
+
+    private ImageCatalogResponse getImageCatalog(UserProfile entity) {
+        ImageCatalog imageCatalog;
+        if (entity.getImageCatalog() == null) {
+            imageCatalog = imageCatalogService.getCloudbreakDefaultImageCatalog();
```

Comment:
Could you please remove `default` image catalog from UserProfileResponse and UserProfileRequest too?